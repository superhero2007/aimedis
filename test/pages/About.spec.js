import React from 'react';
import { shallow } from 'enzyme';

import About from 'pages/About';

function setup() {
  return shallow(<About />);
}

describe('About', () => {
  const wrapper = setup();

  it('should be a Component', () => {
    expect(wrapper.instance() instanceof React.Component).toBe(true);
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
