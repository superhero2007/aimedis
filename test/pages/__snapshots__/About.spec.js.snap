// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render properly 1`] = `"<div class=\\"app__about app__route\\"><div class=\\"container\\"><section class=\\"app__about__section1\\"><div class=\\"app__about__section1__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-6 app__about__section1__content__info\\"><p class=\\"app__about__section1__content__info__title\\">About us</p><p class=\\"app__about__section1__content__info__description\\">Aimedis combines your health data, onliner prescriptions, video consultation with doctors and psychologists, information, medical hardware, your smart devices, and more on a single platform on your PC, your smartphone or your tablet, anytime, worldwide and in multiple languages.</p></div><div class=\\"col-5 offset-1 d-none d-lg-block app__about__section1__content__image\\"><img src=\\"\\" alt=\\"No Asset\\"/><div class=\\"app__about__section1__content__mask\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></div></div></div></section><section class=\\"app__about__section2\\"><div class=\\"app__group__toggle\\"><div class=\\"app__group__toggle__button\\"><div class=\\"app__group__toggle__button__team active\\"><span>TEAM</span></div><div class=\\"app__group__toggle__button__board\\"><span>BOARD</span></div></div><div class=\\"app__group__toggle__content\\"><div class=\\"row\\"><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"/></a><a href=\\"\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"/></a></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"/></a><a href=\\"\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"/></a></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"/></a><a href=\\"\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"/></a></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"/></a><a href=\\"\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"/></a></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"/></a><a href=\\"\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"/></a></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"/></a><a href=\\"\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"/></a></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__image\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><p class=\\"app__group__toggle__item__title\\">Dr. Michael J. Kaldasch</p><p class=\\"app__group__toggle__item__subtitle\\">CEO, MD &amp; founder</p><div class=\\"app__group__toggle__item__link\\"><a href=\\"\\" class=\\"app__group__toggle__item__link__facebook\\"><img src=\\"\\" alt=\\"No Asset\\"/></a><a href=\\"\\" class=\\"app__group__toggle__item__link__linkedin\\"><img src=\\"\\" alt=\\"No Asset\\"/></a></div></div></div></div><div class=\\"col-6 col-lg-3\\"><div class=\\"app__group__toggle__item\\"><div class=\\"app__group__toggle__item__content\\"><div class=\\"app__group__toggle__item__last\\"><div class=\\"app__group__toggle__item__last__image\\"><img src=\\"\\" alt=\\"No asset\\"/></div><p class=\\"app__group__toggle__item__last__title\\">Join our team</p><a href=\\"/career\\" class=\\"btn btn__secondary app__group__toggle__item__last__subtitle\\">SEE ALL POSITIONS</a></div></div></div></div></div></div></div></section><section class=\\"app__about__section3\\"><div class=\\"app__about__section3__title text-center\\"><div class=\\"app__title\\"><p class=\\"app__title__subtitle\\">We are here for you</p><p class=\\"app__title__title\\">Have a questions? Contact us!</p></div></div><div class=\\"app__about__section3__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-6 app__about__section3__content__form\\"><label>Name or Company</label><input type=\\"text\\" class=\\"app__about__section3__content__form__name\\"/><label>E-mail address</label><input type=\\"email\\" class=\\"app__about__section3__content__form__email\\"/><label>Message</label><input type=\\"text\\" class=\\"app__about__section3__content__form__message\\"/><button type=\\"button\\" class=\\"btn btn__primary app__about__section3__content__form__button\\">SEND</button></div><div class=\\"col-lg-6 app__about__section3__content__telegram\\"><p class=\\"app__about__section3__content__telegram__title\\">Official Aimedis Support Group is on <span>Telegram</span></p><a href=\\"#\\" class=\\"app__about__section3__content__telegram__link\\">View channel</a><img class=\\"app__about__section3__content__telegram__image\\" src=\\"\\" alt=\\"No Asset\\"/></div></div></div></section></div><section class=\\"app__about__section4\\"><div style=\\"height:100%\\"></div><div class=\\"container\\"><div class=\\"row\\"><div class=\\"app__about__section4__content\\"><div class=\\"app__about__section4__content__info\\"><p class=\\"app__about__section4__content__info__title\\">OUR LOCATION</p><p class=\\"app__about__section4__content__info__subtitle\\">Sint MichaÃ«lstraat 4,<br/>5935 BL Steyl, Netherlands</p><a href=\\"#\\" class=\\"app__about__section4__content__info__link\\">See on google maps</a></div></div></div></div></section></div>"`;
